src/index.spec.ts

```diff
@@ -2,26 +2,35 @@ import http from "http";
 import nock from "nock";
 
 import { Context, Handler } from "aws-lambda";
+
```

Comment:
generally we try not to introduce new whitespace changes unless the original violated whitespace rules